package-lock.json

```diff
@@ -0,0 +1,4637 @@
+{
```

Comment:
Maybe we can keep this for a separate PR?
This would then have to be a yarn lockfile.